app/src/main/java/com/github/se/gomeet/ui/mainscreens/EventInfo.kt

```diff
@@ -0,0 +1,308 @@
+            title = {
+              // Empty title since we're placing our own components
+            },
+            navigationIcon = {
+              IconButton(onClick = { nav.goBack() }) {
+                Icon(
+                    imageVector = Icons.AutoMirrored.Filled.ArrowBack,
+                    contentDescription = "Back",
+                    tint = MaterialTheme.colorScheme.onBackground)
+              }
+            },
+            actions = {
+              IconButton(onClick = { /* Handle share action */}) {
+                Icon(
+                    imageVector =
+                        ImageVector.vectorResource(id = R.drawable.baseline_chat_bubble_outline_24),
+                    contentDescription = "Share",
+                    modifier = Modifier.size(24.dp),
+                    tint = MaterialTheme.colorScheme.onBackground)
+              }
+
+              IconButton(onClick = { /* Handle more action */}) {
+                Icon(
+                    imageVector = Icons.Filled.MoreVert,
+                    contentDescription = "More",
+                    modifier = Modifier.rotate(90f),
+                    tint = MaterialTheme.colorScheme.onBackground)
+              }
+            })
+      },
+      bottomBar = {
+        // Your bottom bar content
+      }) { innerPadding ->
+        Column(
+            modifier =
+                Modifier.padding(innerPadding)
+                    .padding(start = 15.dp, end = 15.dp, top = 0.dp, bottom = 15.dp)
+                    .fillMaxSize()
+                    .verticalScroll(state = rememberScrollState())) {
+              EventHeader(
+                  title = title,
+                  organizer = organizer,
+                  rating = rating,
+                  nav = nav,
+                  date = date,
+                  time = time)
+              Spacer(modifier = Modifier.height(20.dp))
+              EventButtons()
+              Spacer(modifier = Modifier.height(20.dp))
+              EventImage(painter = image)
+              Spacer(modifier = Modifier.height(20.dp))
+              EventDescription(text = description)
+              Spacer(modifier = Modifier.height(20.dp))
+              MapViewComposable(loc = loc)
+            }
+      }
+}
+
+@Composable
+fun EventHeader(
+    title: String,
+    organizer: String,
+    rating: Double,
+    nav: NavigationActions,
+    date: String,
+    time: String
+) {
+  Row(
+      modifier = Modifier.fillMaxWidth(),
+      verticalAlignment = Alignment.CenterVertically,
+      horizontalArrangement = Arrangement.SpaceBetween) {
+        Column {
+          Text(
+              text = title,
+              style = TextStyle(fontSize = 24.sp, fontWeight = FontWeight.Bold, color = DarkCyan))
+          Spacer(modifier = Modifier.height(5.dp))
+          Text(
+              modifier = Modifier.clickable { nav.navigateTo(SECOND_LEVEL_DESTINATION.get(0)) },
+              text = organizer,
```

Comment:
It seems that we're using the Roboto font for the TextStyle here. To maintain consistency with the fonts used in other screens of the application, should we switch to the standard font?

Reply:
the roboto font isn't working when using high font weights. used it in the description, but it is not used in any title in the app yet. need to find a fix.